v2/test/prepend_testcerts_openssl.sh

```diff
@@ -1,4 +1,40 @@
 #!/bin/bash
-for f in ../testlint/testCerts/*; do
-    openssl x509 -in $f -text -noout | cat - $f > /tmp/out && mv /tmp/out $f
+
+set -e -o pipefail
+
+BASE_DIR=$(dirname "$0")
+CERTS_DIR="$BASE_DIR/../testdata"
+TMP_DIR=$(mktemp -d -t zlint-XXXX)
+
+# Trap EXIT to cleanup the TMP_DIR
+trap '{ rmdir --ignore-fail-on-non-empty $TMP_DIR; }' EXIT
+
+# For every .pem file in the $CERTS directory, prepend 0penSSL text output if
+# required.
+for f in "$CERTS_DIR"/*.pem; do
+  # Skip any files that don't begin with a PEM header. These are assumed to
+  # already have the OpenSSL text output prepended.
+  if [[ ! $(head -n1 "$f") =~ "-----BEGIN" ]]; then
```

Comment:
Just noting:

I was originally going to suggest it might be useful to have a --force option or the like, to force the header to be recomputed even for existing items, if/as more fields are supported (e.g. to make it more useful)

That said, I think I talked myself out of it, because the simplicity of line 25 is worth preserving, and if/when that happens, this 'could' just be a simple bash loop to do the same. That is, it's more likely to be a one-off, so there's no point in building that functionality into this tool and complicating this tool.

Reply:
Good point about wanting to update the text in the future. I'm also happy to defer that to the day when the need presents itself.